src/main/java/duke/Task.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+/**
+ * This class encapsulates the attributes, states and capabilities
+ * of a task: its name, its done status and the ability to change it.
+ *
+ * @author Siau Wee
+ *
+ */
+public class Task {
```

Comment:
```suggestion
public abstract class Task {
```
Suggestion of the `abstract` modifier as the Task is not directly created in Duke.